taxi/views.py

```diff
@@ -34,6 +35,24 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    fields = "__all__"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    fields = "__all__"
+
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Manufacturer
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_delete.html"
```

Comment:
I think we don't need `template_name` here, because we use it only when our first part(before `_delete`) consists of 2 parts, like in Danya's videos `literary_format`, so it will work without `template_name` 
And your Create/Update views are still working without it

Reply:
good then)